src/test/java/juja/sqlcmd/DatabaseManagerTest.java

```diff
@@ -1,66 +1,159 @@
-    public void testConnectWithAllValidParameters() {
-        assertTrue(databaseManager.connect("sqlcmd", "sqlcmd", "sqlcmd"));
+    public void connectWithAllValidParameters() {
+        assertTrue(databaseManager.connect(TEST_DB_NAME, DB_USER_NAME, DB_USER_PASSWORD));
     }
 
     @Test
-    public void testConnectWithInvalidPassword() {
-        assertFalse(databaseManager.connect("sqlcmd", "sqlcmd", "wrongPass"));
+    public void connectWithInvalidPassword() {
+        assertFalse(databaseManager.connect(TEST_DB_NAME, DB_USER_NAME, "wrongPass"));
     }
 
     @Test
-    public void testConnectWithInvalidUserName() {
-        assertFalse(databaseManager.connect("sqlcmd", "noSqlcmd", "sqlcmd"));
+    public void connectWithInvalidUserName() {
+        assertFalse(databaseManager.connect(TEST_DB_NAME, "noSqlcmd", DB_USER_PASSWORD));
     }
 
     @Test
-    public void testConnectWithInvalidDbName() {
-        assertFalse(databaseManager.connect("doesNotExist", "sqlcmd", "sqlcmd"));
+    public void connectWithInvalidDbName() {
+        assertFalse(databaseManager.connect("doesNotExist", DB_USER_NAME, DB_USER_PASSWORD));
     }
 
     @Test
-    public void testInitDatabaseManagerWithInvalidJdbcDriver() {
+    public void initDatabaseManagerWithInvalidJdbcDriver() {
         databaseManager = new DatabaseManager("noDriver", "jdbc:postgresql://localhost:5432/");
-        assertFalse(databaseManager.connect("sqlcmd", "sqlcmd", "sqlcmd"));
+        assertFalse(databaseManager.connect(TEST_DB_NAME, DB_USER_NAME, DB_USER_PASSWORD));
     }
 
     @Test
-    public void testInitDatabaseManagerWithInvalidJdbcUrl() {
+    public void initDatabaseManagerWithInvalidJdbcUrl() {
         databaseManager = new DatabaseManager("org.postgresql.Driver", "noJdbcUrl");
-        assertFalse(databaseManager.connect("sqlcmd", "sqlcmd", "sqlcmd"));
+        assertFalse(databaseManager.connect(TEST_DB_NAME, DB_USER_NAME, DB_USER_PASSWORD));
+    }
+
+    @Test
+    public void getTableNamesWith2tables() throws SQLException {
+        executeSqlQuery("CREATE TABLE table1()");
+        executeSqlQuery("CREATE TABLE table2()");
+        String[] expectedArray = {"table1", "table2"};
+        String[] actualArray = databaseManager.getTableNames();
+        assertArrayEquals(expectedArray, actualArray);
+    }
+
+    @Test
+    public void getTableNamesWithEmptyDb() throws SQLException {
+        String[] expectedArray = new String[]{};
+        String[] actualArray = databaseManager.getTableNames();
+        assertArrayEquals(expectedArray, actualArray);
+    }
+
+    @Test
+    public void getTableDataWithEmptyTable() throws SQLException {
+        executeSqlQuery("CREATE TABLE test_table()");
+        DataSet[] expected = new DataSet[0];
+        assertArrayEquals(expected, databaseManager.getTableData("test_table"));
+    }
+
+    @Test
+    public void getTableDataWithNotExistingTable() throws SQLException {
+        DataSet[] expected = new DataSet[0];
```

Comment:
You named variable expected and two methods above expectedArray.
Try to name similar